.github/actions/install-dependencies/action.yaml

```diff
@@ -10,9 +10,9 @@ runs:
     - uses: actions/setup-go@v3
       with:
         go-version: '1.19'
-    - uses: azure/setup-kubectl@v1
+    - uses: azure/setup-kubectl@v3.0
```

Comment:
Same comment as before. Why `v3.0` and not just `v3`?

Reply:
Good catch! I have updated it.